bitmap-raziyehLogan/model/color.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
```

Comment:
@loganlsr @Raziyehbazargan - got an extra space here - outside of that, this module looks great